static/components/Map.js

```diff
@@ -170,14 +184,32 @@ const Map = (props) => {
         }}
       />
     );
-
+    //current location marker parent
     const currentLocationMarker = currentGPSLocation ? (
       <>
         {translucentBackgroundCircleCurrentLocationMarker}
         {blueFrontCircleCurrentLocationMarker}
       </>
     ) : null;
 
+    //colorLegend
+    const colorLegend = (
+      <div style={colorLegendStyle}>
+        <div style={{ display: 'flex', alignItems: 'center' }}>
+          <div style={{ width: '10px', height: '10px', backgroundColor: 'red', display: 'inline-block', marginRight: '5px', fontStyle: 'bold' }}></div>
+          <span>Lower Than Average</span>
```

Comment:
seems like all three of these can be created in a loop since they are copies of each other with only the color changing